pkgs/applications/office/gtg/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, python3Packages
+, pkgconfig
+, ninja
+, gtk3
+, wrapGAppsHook
+, glib
+, itstool
+, gettext
+, pango
+, gdk-pixbuf
+, gobject-introspection
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gtg";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+      owner = "getting-things-gnome";
+      repo = "gtg";
+      rev = "6623731f301c1b9c7b727e009f4a6462ad381c68";
```

Comment:
This does not match the version.

Reply:
Oops, I guess I was a little late last night, would that be okay?

```nix
let
  rev = "6623731f301c1b9c7b727e009f4a6462ad381c68";
in
python3Packages.buildPythonApplication rec {
  pname = "gtg";
  version = "git-${rev}";

  src = fetchFromGitHub {
    owner = "getting-things-gnome";
    repo = "gtg";
    inherit rev;
```

Reply:
Why not use the 0.4.0 version? Otherwise follow https://nixos.org/nixpkgs/manual/#sec-package-naming.

Reply:
0.4.0 doesn't work for me and the commit version runs fine. since 0.5 is supposed to be released in the near future anyway, I currently prefer the commit version. The name is now gtg-unstable-2020-08-02.